settings/src/test/java/org/odk/collect/settings/validation/JsonSchemaSettingsValidatorTest.kt

```diff
@@ -21,9 +16,15 @@ class JsonSchemaSettingsValidatorTest {
                     }
                 }
             }
-            """.byteInputStream()
-        }
+            """
 
+class JsonSchemaSettingsValidatorTest {
+    private val validator = JsonSchemaSettingsValidator {
```

Comment:
I think it would be easier to read if the validator was constructed in the test. Then you can click-navigate to the schema definition directly.